Pyjion/absint.cpp

```diff
@@ -1319,8 +1327,29 @@ void AbstractInterpreter::branch_raise(char *reason) {
             break;
         }
     }
+    return count;
+}
+
+void AbstractInterpreter::ensure_labels(vector<Label>& labels, size_t count) {
+    for (auto i = labels.size(); i < count; i++) {
+        labels.push_back(m_comp->emit_define_label());
+    }
+}
+
+void AbstractInterpreter::branch_raise(char *reason) {
+    auto ehBlock = get_ehblock();
```

Comment:
Yes, I've been kind of all over the place while getting used to C++ again...  We'll need to pick something and then do a broad update of all of the existing code.  In some places we interface with other code (e.g. the JIT interface) so there will likely be some apparent inconsistency :(